src/PowerShellEditorServices.Host/EditorServicesHost.cs

```diff
@@ -70,7 +70,7 @@ public class EditorServicesHost
         private IServerListener languageServiceListener;
         private IServerListener debugServiceListener;
 
-        private TaskCompletionSource<bool> serverCompletedTask;
+        private TaskCompletionSource<bool> serverCompletedTask = new TaskCompletionSource<bool>();

```

Comment:
I see this is also assigned in the constructor. Is there a need to initialise it here as well? Can we initialise it in the constructor instead of here?

Reply:
It's assigned in the constructor of LanguageServer not EditorServicesHost.

I could move the assignment to the constructor of EditorServices as well though - like featureFlags.

Reply:
fixed